### ISSUE TITLE
Why load CARFT and vgg16_bn separately?

### ISSUE THREAD

**Issue by ThisIsIsaac** [clovaai/CRAFT-pytorch#68]
My understanding is that weights for `vgg16_bn` is the pretrained model released by Pytorch and CRAFT is trained with SynthText.

1. Why did you chose to use Pytorch's pretrained model instead of the weights trained together with CRAFT? Does pytorch's perform better?
2. When training, my understanding is that gradient should flow through both, so we should unfreeze both networks, even if we are using Pytorch's pretrained weights for `vgg16_bn`. Am I correct?

**Comment by YoungminBaek** [comment]
@ThisIsIsaac 
1. We trained the CRAFT from the pretrained model released by Pytorch. In summary, the training procedure is (1) image-net pretrained model, (2) pretrain on SynthText, (3) fie-tune on real datasets.
2. That's correct. We did not freeze the backbone network when training.
